y/iterator_test.go

```diff
@@ -75,6 +75,11 @@ func (s *SimpleIterator) Value() ValueStruct {
 		Meta:     0,
 	}
 }
+func (s *SimpleIterator) FillValue(vs *ValueStruct) {
+	vs.Value = s.vals[s.idx]
+	vs.UserMeta = 55
```

Comment:
What is the magic number?

Reply:
Just a test value, the same as the `Value()` method.

Reply:
Please add comments.